packages/mobile/src/home/WalletHome.tsx

```diff
@@ -190,6 +171,14 @@ export class WalletHome extends React.Component<Props> {
     }
   }
 
+  onPressQrCode = () => {
```

Comment:
In some places we name these handlers onPress, in others onClick. Let's make a mental note to pick one later and stay consistent. I'm for onClick, you?

Reply:
I'm for `onPressX` 😈 😄 . Mostly because that's the terminology used by react-native and makes sense on mobile.

Reply:
Ah good point about mobile. Ok onPress then :)